src/main/java/org/benetech/servicenet/security/AuthoritiesConstants.java

```diff
@@ -5,6 +5,8 @@
  */
 public final class AuthoritiesConstants {
 
+    public static final String SACRAMENTO = "SACRAMENTO";
```

Comment:
rest of the authorities have a ROLE_ prefix, I think it should have it too

Reply:
done